tutorlimesurvey/templates/limesurvey/apps/application/config/config.php

```diff
@@ -0,0 +1 @@
+{{ LIMESURVEY_CONFIG }}
```

Comment:
Is an empty config valid?

Reply:
I check it it's empty so we don't have to render it